recipe/meta.yaml

```diff
@@ -14,8 +14,8 @@ package:
 
 source:
   # latest commit from branch {{ cctools_version }}-ld64-{{ ld64_version }}
-  - url: https://github.com/tpoechtrager/cctools-port/archive/437ced391dbf14dce86f977ca050a750d5682f39.tar.gz
-    sha256: cf4020ddb92467de020c322eb13f0cac9ed992daa862a09bc2607e42467b3042
+  - url: https://github.com/tpoechtrager/cctools-port/archive/2a3e1c2a6ff54a30f898b70cfb9ba1692a55fad7.tar.gz
+    sha256: b1c1b148e14c3dacf99018a519506a26424c635d24ab6820c20b21e5ca22b8a3
```

Comment:
Please use the commit from https://github.com/tpoechtrager/cctools-port/tree/1009.2-ld64-907 and update the version. If you need a specific later, add it to the patches below

Reply:
Are we good to update the version? Happy to do that - but don't know if there is a migration involved.

Reply:
Since it turned out to be quite some more work to update to the latest version, I've added the patch and bumped the build number. Do you want to merge your stdlib fixes into this PR?